server/darkmoon/settings.py

```diff
@@ -1,17 +1,47 @@
 """Statement imports libraries."""
+
+###########
+# IMPORTS #
+###########
+
+import os
+
+from dotenv import load_dotenv
 from pydantic import BaseSettings
 
+#############
+# FUNCTIONS #
+#############
+
+load_dotenv()
+
+###########
+# CLASSES #
+###########
+
 
 class Settings(BaseSettings):
     """Settings class."""
 
-    DATABASE_URL: str = "mongodb://darkmoon:password@172.16.5.3/"
```

Comment:
Backing what Tory said. Also the fields in the Settings class should be a 1 to 1 match from your .env

You can see in the main branch how we have `DATABASE_URL` in '.env` and in our Setting model


Here's their Settings model example: [https://docs.pydantic.dev/latest/usage/settings/](url)